code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -632,6 +632,8 @@
 /datum/species/golem/clockwork/no_scrap //These golems are created through the herald's beacon and leave normal corpses on death.

```

Comment:
These coulda just not been golems honestly

Reply:
Mumble grumble consistency

Reply:
They're gonna get messages from adamantium ones and stuff as well